### ISSUE TITLE
Is there official support/no support for Sets?

### ISSUE THREAD

**Issue by antonytrupe** [Belphemur/node-json-db#721]
I swear I read before something about Sets not working, but I'm striking out finding it again. Is it supposed to handle Sets, or is that out of scope currently? It looks like it saves them as empty objects.

**Comment by Belphemur** [comment]
Currently, it doesn't support Set, Map etc ...

For that we'd need to write/update the custom json serializer and deserializer to support those type

**Comment by antonytrupe** [comment]
Right, of course. Is that something that fits in the design goals for this project, or not so much?

I'm currently using arrays/objects and then converting them as as they cross the memory/store boundary, which doesn't seem wrong.
